docs/docs-beta/docs/getting-started/quickstart.md

```diff
@@ -0,0 +1,177 @@
+---
+title: "Dagster quickstart"
+description: Learn how to quickly get up and running with Dagster
+sidebar_position: 30
+sidebar_label: "Quickstart"
+---
+
+# Build your first Dagster project
+
+Welcome to Dagster! In this guide, you'll use Dagster to create a basic Extract, Transform, Load (ETL) pipeline that extracts data from a CSV file and transforms it.
+
+## What you'll learn
+
+- How to set up a basic Dagster project
+- How to create a Dagster asset for each step of the ETL process
+- How to use Dagster's built-in features to monitor and execute your pipeline
+
+## Prerequisites
+
+<details>
+  <summary>Prerequisites</summary>
+
+To follow the steps in this guide, you'll need:
+
+- Basic Python knowledge
+- Python 3.8+ installed on your system. Refer to the [Installation guide](/getting-started/installation) for information.
+</details>
+
+## Step 1: Set up the Dagster environment
+
+1. Open the terminal and create a new directory for your project:
+
+   ```bash
+   mkdir dagster-quickstart
+   cd dagster-quickstart
+   ```
+
+2. Create and activate a virtual environment:
+
+   <Tabs>
+   <TabItem value="macos" label="MacOS">
+   ```bash
+   python -m venv venv
+   source venv/bin/activate
+   ```
+   </TabItem>
+   <TabItem value="windows" label="Windows">
+   ```bash
+   python -m venv venv
+   source venv\Scripts\activate
+   ```
+   </TabItem>
+   </Tabs>
+
+3. Install Dagster and the required dependencies:
+
+   ```bash
+   pip install dagster dagster-webserver pandas
+   ```
+
+## Step 2: Create the Dagster project structure
+
+:::info
+The project structure in this guide is simplified to allow you to get started quickly. As a next step, check out the [ETL pipeline tutorial](/tutorial/tutorial-etl) to learn how to build more complex pipelines using best practices.
+:::
+
+Next, you'll create a basic Dagster project that looks like this:
+
+```
+dagster-quickstart/
+├── quickstart/
+│   ├── __init__.py
+│   └── assets.py
+├── data/
+    └── sample_data.csv
+```
+
+1. To create the files and directories outlined above, run the following:
+
+   ```bash
+   mkdir quickstart data
+   touch quickstart/__init__.py quickstart/assets.py
+   touch data/sample_data.csv
+   ```
+
+2. In the `data/sample_data.csv` file, add the following content:
+
+   ```csv
+   id,name,age,city
+   1,Alice,28,New York
+   2,Bob,35,San Francisco
+   3,Charlie,42,Chicago
+   4,Diana,31,Los Angeles
+   ```
+
+   This CSV will act as the data source for your Dagster pipeline.
+
+## Step 3: Define the assets
+
+Now, create the assets for the ETL pipeline. Open `quickstart/assets.py` and add the following code:
+
+```python
```

Comment:
Will we move to code example in `examples/docs_snippets` for proper validation?

Reply:
Yes! I'll take care of that, ty